ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -0,0 +1,45 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy.functional.backends.torch as ivy_torch
+
+
+# flip
+@given(
+    dtype_values_axis = helpers.dtype_values_axis(

```

Comment:
Please use `dtype_and_values` and `get_axis` separately for the reasons specified in the previous review.

Reply:
on a seperate note: nested helper strategies doesn't work right? as in:
```
@given(
axis = helpers.get_axis(shape=helpers.get_shape())
)
```
the above doesn't work right? b/c get_shape doesn't eval in the @given() decorator for get_axis to eval?

Reply:
does this argument work?
```
axis = helpers.get_axis(shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"), ret_tuple=True),
```